packages/playback-core/src/index.ts

```diff
@@ -66,6 +77,8 @@ export type MuxMediaPropTypes = {
     | `${Uppercase<keyof MimeTypeShorthandMap>}`;
   streamType: ValueOf<StreamTypes>;
   startTime: HlsConfig["startPosition"];
+  autoPlay: boolean | ValueOf<AutoplayTypes>;
+  autoplay: boolean | ValueOf<AutoplayTypes>;
```

Comment:
I'm not sure if there's a way to combine these or eliminate the duplication here.
It's likely possible if we wanted to use `autoPlay` only and then reference to it as `props.autoPlay` below on lines 319-441

Reply:
yeah there probably is (or at least, moving the definition relationships around), but I don't think we need to prioritize that now... may lead to some "lying" hints/false positives, but I can pull on the definitional threads as a followup.